gitops/applications/overlays/cloud_provider/aws/xplane-provider-config/composition-rds-aurora.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: rds-aurora-composition
+  labels:
+    provider: aws
+    db: rds-aurora
+    cluster: ${ARGOCD_ENV_cluster_name}
+spec:
+  compositeTypeRef:
+    apiVersion: infitx.org/v1alpha1
+    kind: RDSAuroraDBCluster
+  mode: Pipeline
+  pipeline:
+  - functionRef:
+      name: function-patch-and-transform
+    step: patch-and-transform
+    input:
+      apiVersion: pt.fn.crossplane.io/v1beta1
+      kind: Resources
+      patchSets:
+      - name: metadata
+        patches:
+        - fromFieldPath: metadata.annotations
+          toFieldPath: metadata.annotations
+        - fromFieldPath: metadata.labels
+          toFieldPath: metadata.labels          
+        - fromFieldPath: spec.id
+          toFieldPath: metadata.name
+      resources:
+      - name: dbsubnetgroup
+        base:
+          apiVersion: database.aws.crossplane.io/v1beta1
+          kind: DBSubnetGroup
+          spec:
+            forProvider:
+              description: Subnet group for DB instance
+              region: eu-west-2
```

Comment:
why is the region hard coded?  do you have to set here in order to patch later?  if so, pick an unusable value si it is obvious.

Reply:
- fromFieldPath: "spec.parameters.region"
          toFieldPath: "spec.forProvider.region"
   Since the above patch is present , it will surely get overridden by the managed resource definition.

Reply:
lit will get patches for sure, but the code leaves an impression that you are setting an actual region to eu-west-2, if you set it to TEMP-PLACEHOLDER for example, it is clear that the patch will override.

Reply:
ok

Reply:
Please find the changed code 
https://github.com/mojaloop/iac-modules/pull/430/commits/a99bf5d3754a6674a56d3e330179367a8b18f975